tests/test_model.py

```diff
@@ -1,6 +1,7 @@
+import sys
```

Comment:
also here, can we sort the imports and separate standard library from third party ; so here ```import functools``` should be the first one and the newline before the ```numpy``` imports should stay